.github/workflows/main.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test build env
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        lfs: true
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    - name: Test
+      shell: 'script -q -e -c "bash {0}"'
+      run: |
+        mkdir -p /tmp/test-rpm-build
+        git clone https://github.com/xcp-ng-rpms/test-rpm-build.git /tmp/test-rpm-build
+        GIT_DESCRIBE=$(git describe --tags)
+        VERSION=${GIT_DESCRIBE:0:3}
```

Comment:
Add a special case for master branch?

Currently a build towards master would use VERSION=8.2 but in the future this will change to 8.3 (and since this will change, this should be defined only once in the test-rpm-build repo. Another reason to define the workflow there)

Reply:
If the branch are merged into master the tag will work on master, 8.2 until 8.3 is merged into master.

Reply:
But I like the reusable worklow indeed.

Reply:
At some point, before the final release of 8.3 (thus before the tag), we will need to use a 8.3 build env for master branch.